### ISSUE TITLE
Collaborate with TOC to clarify and publish TAG leadership role requirements for O11y TAG leaders

### ISSUE THREAD

**Issue by alolita** [cncf/tag-observability#103]
The TAG leadership role, responsibilities and participation requirements need to be clearly defined in the CNCF [TAG charter ](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) doc. 

Action: The TAG leadership will work with the TOC to clarify and publish these requirements.

**Comment by halcyondude** [comment]
I agree that [current definitions](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#tag-member-roles) are missing some of these things.  Yesterday the TOC and TAG Chairs met and discussed this and other topics.  It was a good discussion and I think that we'll see fruit. To raise awareness around this, here's a link to the meeting: https://www.youtube.com/watch?v=79fFiOO7XzQ&t=889s, https://sched.co/1T8tH

**Comment by alolita** [comment]
Bumping this thread up for discussion and definition. 

@halcyondude - let's work on this.
